auslib/test/admin/views/test_releases.py

```diff
@@ -215,8 +225,8 @@ def testDeleteNonExistentRelease(self):
         self.assertStatusCode(ret, 404)
 
     def testDeleteWithoutPermission(self):
-        ret = self._delete("/releases/a", username="bob")
-        self.assertStatusCode(ret, 401)
+        ret = self._delete("/releases/a", username="bob", qs=dict(data_version=1))
+        self.assertStatusCode(ret, 403)
```

Comment:
This looks to be blocked by bob's permissions to modify products 'fake' and 'b' only. Could we have a test based on actions too ?